### PR TITLE
:rocket: feat: Added review-attachments zoomer

### PR DESCRIPTION
**Pull request by shivendra-webkul**
## Issue Reference
Added review-image zoomer component

### CODE REVIEW

File: packages/Webkul/Shop/src/Resources/views/products/view/gallery.blade.php [hunk 1]
```diff
@@ -79,7 +79,7 @@ class="min-w-[450px] cursor-pointer rounded-xl"
                         alt="{{ $product->name }}"
                         width="560"
                         height="610"
-                        @click="$emitter.emit('v-show-images-zoomer', activeIndex)"
+                        @click="isImageZooming =! isImageZooming"
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/src/Resources/views/products/view/gallery.blade.php:82]: Fix this.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/Shop/src/Resources/views/products/view/gallery.blade.php (hunk 1 above)
Comment (shivendra-webkul) [reply packages/Webkul/Shop/src/Resources/views/products/view/gallery.blade.php]: Done

**Review by devansh-webkul — APPROVED** [review]

**Review by devansh-webkul — DISMISSED** [review]

**Comment by ashishkumar-webkul** [comment]
Checked and Found that Images or videos in vertical size are not appearing properly, which need to be managed.

**Video**

https://github.com/bagisto/bagisto/assets/92721837/4a8feded-09ef-4057-b1d6-c3e92e71b5b0

**Comment by ashishkumar-webkul** [comment]
Checked and Found that  the images and banner both are appearing clear in the review zoomer. Please check  the video for the reference.

**Video**

https://github.com/bagisto/bagisto/assets/92721837/67a09895-ca0e-426c-9417-f51e363a8381

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**